lib/resque/scheduler/job.rb

```diff
@@ -0,0 +1,40 @@
+# vim:fileencoding=utf-8
+
+module Resque
+  module Scheduler
+    module Job
+      class << self
+        def included(base)
+          base.extend ClassMethods
+        end
+      end
+
+      module ClassMethods
+        def cron(value = nil)
+          return @cron ||= nil if value.nil?
+          @cron = value
+        end
+
+        def every(value = nil)
+          return @every ||= nil if value.nil?
+          @every = value
+        end
+
+        def queue(value = nil)
+          return @queue ||= nil if value.nil?
+          @queue = value
+        end
```

Comment:
The standard Resque convention is to use `@queue` without a class method. I think we should continue to use that convention if we can.

Reply:
You are right. Fixed.